app/grandchallenge/components/apps.py

```diff
@@ -41,6 +41,66 @@ def init_default_interfaces(*_, **__):
             "kind": ComponentInterface.Kind.JSON,
             "relative_path": "metrics.json",
         },
+        {
+            "title": "Boolean",
+            "kind": ComponentInterface.Kind.BOOL,
+            "relative_path": "bool",
+        },
+        {
+            "title": "String",
+            "kind": ComponentInterface.Kind.STRING,
+            "relative_path": "string",
+        },
+        {
+            "title": "Integer",
+            "kind": ComponentInterface.Kind.INTEGER,
+            "relative_path": "int",
+        },
+        {
+            "title": "Float",
+            "kind": ComponentInterface.Kind.FLOAT,
+            "relative_path": "float",
+        },
+        {
+            "title": "2D bounding box",
+            "kind": ComponentInterface.Kind.TWO_D_BOUNDING_BOX,
+            "relative_path": "2d_bounding_box",
+        },
+        {
+            "title": "Multiple 2D bounding boxes",
+            "kind": ComponentInterface.Kind.MULTIPLE_TWO_D_BOUNDING_BOXES,
+            "relative_path": "multiple_2d_bounding_boxes",
+        },
+        {
+            "title": "Distance measurement",
+            "kind": ComponentInterface.Kind.DISTANCE_MEASUREMENT,
+            "relative_path": "distance_measurement",
+        },
+        {
+            "title": "Multiple distance measurements",
+            "kind": ComponentInterface.Kind.MULTIPLE_DISTANCE_MEASUREMENTS,
+            "relative_path": "multiple_distance_measurements",
+        },
+        {
+            "title": "Point",
+            "kind": ComponentInterface.Kind.POINT,
+            "relative_path": "point",
+        },
+        {
+            "title": "Multiple points",
+            "kind": ComponentInterface.Kind.MULTIPLE_POINTS,
+            "relative_path": "multiple_points",
+        },
+        {
+            "title": "Polygon",
+            "kind": ComponentInterface.Kind.POLYGON,
+            "relative_path": "polygon",
+        },
+        {
+            "title": "Multiple polygons",
+            "kind": ComponentInterface.Kind.MULTIPLE_POLYGONS,
+            "relative_path": "multiple_polygons",
+        },
```

Comment:
These will end up in the production database. Do we really want those with these default values? Isn't it better to add these as fixtures in `init_gc_demo` or as factories in the tests?